src/jira/client/index.ts

```diff
@@ -213,7 +213,46 @@ async function getJiraClient(
 					);
 				}
 			}
-		}
+		},
+		workflow: {
+			submit: async (data) => {
+				updateIssueKeysFor(data.builds, dedup);
+				if (!withinIssueKeyLimit(data.builds)) {
+					updateIssueKeysFor(data.builds, truncate);
+					const subscription = await Subscription.getSingleInstallation(jiraHost, gitHubInstallationId);
+					await subscription.update({ syncWarning: "Exceeded issue key reference limit. Some issues may not be linked." });
+				}
+				const payload = {
+					builds: data.builds,
+					properties: {
+						gitHubInstallationId,
+					},
+					providerMetadata: {
+						product: data.product,
+					},
+				};
+				logger.info(`Sending builds payload to jira. Payload: ${payload}`);
+				await instance.post("/rest/builds/0.1/bulk", payload);
+			},
+		},
+		deployment: {
+			submit: async (data) => {
+				updateIssueKeysFor(data.deployments, dedup);
+				if (!withinIssueKeyLimit(data.deployments)) {
+					updateIssueKeysFor(data.deployments, truncate);
+					const subscription = await Subscription.getSingleInstallation(jiraHost, gitHubInstallationId);
+					await subscription.update({ syncWarning: "Exceeded issue key reference limit. Some issues may not be linked." });
+				}
+				const payload = {
+					deployments: data.deployments,
+					properties: {
+						gitHubInstallationId,
+					},
+				};
+				logger.info(`Sending deployments payload to jira. Payload: ${payload}`);
```

Comment:
I don't think we should log payloads hence they contain UGC. If we need it for testing we should change to `debug`

- [ ] remove payloads logging in prod

Reply:
Still good to have a logline which indicates that the request was made. So we just need to remove the ${payload} part